outputs.tf

```diff
@@ -81,6 +81,11 @@ output "oidc_provider_arn" {
   value       = var.enable_irsa ? concat(aws_iam_openid_connect_provider.oidc_provider[*].arn, [""])[0] : null
 }
 
+output "oidc_provider_url" {
+  description = "The URL of the OIDC Provider if `enable_irsa = true`."
+  value       = var.enable_irsa ? concat(aws_iam_openid_connect_provider.oidc_provider[*].url, [""])[0] : null
+}
+
```

Comment:
Is this different from `aws_eks_cluster.this[*].identity[*].oidc.0.issuer`

```hcl
output "cluster_oidc_issuer_url" {
  description = "The URL on the EKS cluster OIDC Issuer"
  value       = flatten(concat(aws_eks_cluster.this[*].identity[*].oidc.0.issuer, [""]))[0]
}
```

Because, it sounds that the `aws_iam_openid_connect_provider.oidc_provider.url` is the same. See https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/irsa.tf#L14

Reply:
I have checked:
`cluster_oidc_issuer_url` producing: `https://oidc.eks.us-east-1.amazonaws.com/id/79867GSDBFCCA9C8877335526D5DC38` 
`aws_iam_openid_connect_provider.oidc_provider[*].url` just `oidc.eks.us-east-1.amazonaws.com/id/79867GSDBFCCA9C8877335526D5DC38` 

ALB integration which expects full URL:
```
....
    authenticate_oidc {
      authorization_endpoint = "https://example.com/authorization_endpoint"
      client_id              = "client_id"
      client_secret          = "client_secret"
      issuer                 = "https://example.com"
      token_endpoint         = "https://example.com/token_endpoint"
      user_info_endpoint     = "https://example.com/user_info_endpoint"
    }
....
```
so in general `cluster_oidc_issuer_url` can be used directly into that integration. 

So closing this PR